enhancements/audit-command.md

```diff
@@ -0,0 +1,594 @@
+$ tar -xvf mybundle.tar 
+$ tree
+.
+├── 696782e86a62476638704177b71dd37382864a1801866002cd6628d1a3eec4c0
+│   ├── VERSION
+│   ├── json
+│   └── layer.tar
+├── f6bbe84c78f5d0725d248207f92a23edfcbe66d3016f841018c03f933341dae3.json
+├── manifest.json
+└── mybundle.tar
+$ 
+``` 
+
+Now, see that all manifests are in the `layer.tar`:
+
+```
+$ tar -xvf 696782e86a62476638704177b71dd37382864a1801866002cd6628d1a3eec4c0/layer.tar
+$ tree
+.
+├── 696782e86a62476638704177b71dd37382864a1801866002cd6628d1a3eec4c0
+│   ├── VERSION
+│   ├── json
+│   └── layer.tar
+├── f6bbe84c78f5d0725d248207f92a23edfcbe66d3016f841018c03f933341dae3.json
+├── manifest.json
+├── manifests
+│   ├── myopertor.v0.5.5.clusterserviceversion.yaml
+│   ├── apps_v1alpha1_apimanager_crd.yaml
+│   ├── capabilities_v1alpha1_api_crd.yaml
+│   ├── capabilities_v1alpha1_binding_crd.yaml
+│   ├── capabilities_v1alpha1_limit_crd.yaml
+│   ├── capabilities_v1alpha1_mappingrule_crd.yaml
+│   ├── capabilities_v1alpha1_metric_crd.yaml
+│   ├── capabilities_v1alpha1_plan_crd.yaml
+│   └── capabilities_v1alpha1_tenant_crd.yaml
+├── metadata
+│   └── annotations.yaml
+├── mybundle.tar
+└── root
+    └── buildinfo
+        ├── Dockerfile-myoperator-rhel7-operator-metadata-2.8.2-5
+        └── content_manifests
+            └── myopertor-bundle-container-2.8.2-5.json
+```
+
+#### Using SDK bundle validate
+
+[SDK](https://github.com/operator-framework/operator-sdk) allows us to validate all bundles: 
+
+```
+ $ operator-sdk bundle validate ./bundle --select-optional suite=operatorframework --output=json-alpha1
+
+INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
+INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
+{
+    "passed": false,
+    "outputs": [
+        {
+            "type": "error",
+            "message": "Error: Value : (memcached-operator.v0.0.1) csv.Spec.Icon elements should contain both data and mediatype"
+        }
+    ]
+}
+```
+
+Currently, the validators used by it will only ensure the bundle [spec definition](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md) and the standard requirements to publish an operator in the OperatorHub.io. For further information also check the proposal [operator-framework/api: add new checks to the operatorhub validator][ep-new-checks for-operator-hub].
+
+#### Using SDK scorecard bundle
+
+For the SDK tool be able to run scorecard tests its configuration requires to be wrote in the bundle directory `tests/scorecard/config.yaml`. ([example](https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/tests/scorecard/config.yaml)). Operator bundles which are build with SDK will have the scorecard tests configured by default. ([example](https://github.com/operator-framework/community-operators/tree/master/community-operators/namespace-configuration-operator/1.0.1)).
```

Comment:
While true, there is no official guidance on bundling tests. Plus I don't think scorecard can respect a pullspec, such that scorecard image tags can be substituted and run within a pipeline.

Reply:
Sorry, but I think you might misunderstand the proposal at this point. So, I will try to clarify :

- the command will check - has or not scorecard config files on the bundle
- if not: then scaffold the default tests ( same code that we scaffold by default in any new project built with SDK )
- if yes: do nothing

I will try to clarify that in the proposal as well. Others might have the same doubt.

Reply:
I clarified it better so I think that as fine close this one. Let me know if you still with concerns and feel free to re-open if you see that is required.